<launch>

  <node name="move_group_interface_tutorial" pkg="mobile_arm_moveit_perception_config" type="move_group_interface_tutorial" respawn="false" output="screen">
  </node>

</launch>